debezium-core/src/main/java/io/debezium/connector/SnapshotRecord.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector;
+
+import org.apache.kafka.connect.data.Struct;
+
+/**
+ * Describes whether the change record comes from snapshot and if it is the last one
+ * <ul>
+ * <li>{@link #TRUE} - record is from snapshot is not the last one</li>
+ * <li>{@link #LAST} - record is from snapshot is the last record generated in snapshot phase</li>
+ * <li>{@link #FALSE} - record is from streaming phase</li>
+ * </ul>
+ * 
+ * @author Jiri Pechanec
+ *
+ */
+public enum SnapshotRecord {
+    TRUE,
+    FALSE,
+    LAST;
+
+    public static SnapshotRecord fromSource(Struct source) {
+        if (source.schema().field(AbstractSourceInfo.SNAPSHOT_KEY) != null && io.debezium.data.Enum.LOGICAL_NAME.equals(source.schema().field(AbstractSourceInfo.SNAPSHOT_KEY).schema().name())) {
+            final String snapshotString = source.getString(AbstractSourceInfo.SNAPSHOT_KEY);
+            if (snapshotString != null) {
+                return SnapshotRecord.valueOf(snapshotString.toUpperCase());
+            }
+        }
+        return null;
```

Comment:
Shouldn't it be `FALSE`?

Reply:
@gunnarmorling No, `FALSE` is the default value in the schema and IIRC we agreed we don't want to send `snapshot` for streming phase

Reply:
Ok, so I still think it'd be better to express that default behavior in the enum itself and make the fact, that the value for FALSE is omitted in the source info just a local matter of "rendering" the property there. But it's not too important either.